azerothcore/azerothcore-wotlk
# fix(DB/waypoint_data): Ulduar Colossus path fix

## CHANGES PROPOSED:
Edit waypoints to prevent Ulduar Colossus from clipping beneath the bridge in the Iron Concourse in Ulduar and slightly extend pathing.

## HOW TO TEST THE CHANGES:

Before update:
- .go c 137478
- Observe pathing.

After update:
- Restart server
- Observe new pathing.

## Target branch(es):
- 90a10a9


## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
